test/test.ts

```diff
@@ -4,8 +4,8 @@ import { PinejsClientCore } from '..';
 
 // Create a class for tests that extends the exported abstract class
 class PinejsClient extends PinejsClientCore {
-	public async _request(): Promise<any> {
-		return;
+	public _request(): Promise<any> {
```

Comment:
Alternatively, we can ignore the rule

Reply:
I generally prefer ignoring the rule for cases like this where it needs to be a dummy promise, but also it's tests so I'm not fussed